documentation/book/con-resource-format.adoc

```diff
@@ -0,0 +1,8 @@
+[id='kafka-resource-format{context}']
+= Overview of Kafka Resouce Format
+
+The full API is described in <<type-KafkaAssembly>>.
```

Comment:
Use xref and the standard language for referencing more information ... also I don't know that the first thing we should do is link out to another section here.